opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java

```diff
@@ -69,64 +66,12 @@ public void end(EndSpanOptions options) {}
   public static final String MESSAGE_EVENT_ATTRIBUTE_KEY_SIZE_COMPRESSED =
       "message.event.size.compressed";
 
-  private static final Tracer TRACER =
-      OpenTelemetry.getGlobalTracer("io.opencensus.opentelemetry.migration");
-
   private SpanConverter() {}
 
-  static io.opentelemetry.api.trace.Span toOtelSpan(Span span) {
-    if (span == null) {
-      return io.opentelemetry.api.trace.Span.getInvalid();
-    }
-    SpanData ocSpanData = ((RecordEventsSpanImpl) span).toSpanData();
-    SpanBuilder builder =
-        TRACER
-            .spanBuilder(ocSpanData.getName())
-            .setStartTimestamp(
-                TimeUnit.SECONDS.toNanos(ocSpanData.getStartTimestamp().getSeconds())
-                    + ocSpanData.getStartTimestamp().getNanos(),
-                TimeUnit.NANOSECONDS);
-    if (ocSpanData.getKind() != null) {
-      builder.setSpanKind(mapKind(ocSpanData.getKind()));
-    }
-    if (ocSpanData.getAttributes() != null) {
-      for (Map.Entry<String, AttributeValue> attribute :
-          ocSpanData.getAttributes().getAttributeMap().entrySet()) {
-        attribute
-            .getValue()
-            .match(
-                setStringAttribute(builder, attribute),
-                setBooleanAttribute(builder, attribute),
-                setLongAttribute(builder, attribute),
-                setDoubleAttribute(builder, attribute),
-                arg -> null);
-      }
-    }
-    if (ocSpanData.getLinks() != null) {
-      for (Link link : ocSpanData.getLinks().getLinks()) {
-        AttributesBuilder attributesBuilder = Attributes.builder();
-        link.getAttributes()
-            .forEach(
-                (s, attributeValue) ->
-                    attributeValue.match(
-                        setStringAttribute(attributesBuilder, s),
-                        setBooleanAttribute(attributesBuilder, s),
-                        setLongAttribute(attributesBuilder, s),
-                        setDoubleAttribute(attributesBuilder, s),
-                        arg -> null));
-        builder.addLink(
-            io.opentelemetry.api.trace.SpanContext.create(
-                TraceId.bytesToHex(link.getTraceId().getBytes()),
-                SpanId.bytesToHex(link.getSpanId().getBytes()),
-                TraceFlags.getDefault(),
-                TraceState.getDefault()),
-            attributesBuilder.build());
-      }
+  static Kind mapKind(@Nullable Span.Kind kind) {
```

Comment:
Let's use fullly qualified class names, it's really tricky knowing which Kind is which